docs/Installation.md

```diff
@@ -527,7 +540,7 @@ you should be able to use the
 API on localhost port 5000 as below. The command requires that `curl` is installed.
 
 ```sh
-curl -s -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","method":"version_info","id":"1"}' http://localhost:5000/ && echo

```

Comment:
Both strings and numbers are allowed here by the JSON-RPC specification. If nothing else, using a number is two slightly shorter.